src/modules/facility-guarantee/dto/create-facility-guarantee-request.dto.ts

```diff
@@ -0,0 +1,75 @@
+import { EXAMPLES } from '@ukef/constants';
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { AcbsPartyId, UkefId } from '@ukef/helpers';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+
+export type CreateFacilityGuaranteeRequest = CreateFacilityGuaranteeRequestItem[];
+
+export class CreateFacilityGuaranteeRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The identifier of the facility to create the guarantee for.',
+    example: EXAMPLES.FACILITY_ID,
+    minLength: 10,
+    maxLength: 10,
+    pattern: /^00\d{8}$/,
+  })
+  readonly facilityIdentifier: UkefId;
+
+  @ValidatedDateOnlyApiProperty({
+    description: `The date that this guarantee will take effect. This will be replaced by today's date if a date in the past is provided.`,
+  })
+  readonly effectiveDate: DateOnlyString;
+
+  @ValidatedStringApiProperty({
+    description: 'An ACBS party identifier.',
+    minLength: 8,
+    maxLength: 8,
```

Comment:
Should we just add `length: 8` decorator as it will suffice both min and max.

Reply:
Changed to use length instead minLength and maxLength.